runtime/src/bank.rs

```diff
@@ -2869,11 +2869,34 @@ impl Bank {
         stake_weighted_timestamp
     }
 
+    /// Recalculates the bank hash
+    ///
+    /// This is used by ledger-tool when creating a snapshot, which
+    /// recalcuates the bank hash.
```

Comment:
Just confirming, this is only used by ledger-tool by virtue of this function ?
https://github.com/anza-xyz/agave/blob/598f7aedf98bf6be2d5f6afd5f14e8c6070ab20a/runtime/src/snapshot_bank_utils.rs#L887-L893

And that function must be `pub` in order to allow its' usage for tests in other crates ? Was trying to figure out if we could hide all of this behind dcou, but it is looking like that is not an option